frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,265 @@
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-auth-identity-help"
+                  name="emailAuthIdentity"
+                  data-test-id="email-auth-identity"
+                  value={formValues.emailAuthIdentity}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Secret (CRAM-MDS)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-secret-help"
+                  name="emailAuthSecret"
+                  data-test-id="email-auth-secret"
+                  value={formValues.emailAuthSecret}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+          </div>
+          <label className="co-no-bold">
+            <input
+              type="checkbox"
+              name="emailRequireTls"
+              onChange={(e) =>
+                handleChange({
+                  target: { name: 'emailRequireTls', value: e.target.checked },
+                })
+              }
+              checked={formValues.emailRequireTls}
+            />
+            &nbsp; Require TLS
+          </label>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const getInitialValues = (globals, receiverConfig) => {
+  const getInitialValue = (propName: string) =>
+    _.get(receiverConfig, propName) || _.get(globals, `smtp_${propName}`) || '';
+
+  const initValues: any = { [SAVE_AS_DEFAULT_FLD]: false };
+  initValues.emailTo = _.get(receiverConfig, 'to') || '';
+
+  _.forEach(SMTP_CONFIG_FIELDS, (propName) => {
+    initValues[`${getFormFieldName(FORM_PREFIX, propName)}`] = getInitialValue(propName);
+  });
+
+  return initValues;
+};
+
+export const isFormInvalid = (formValues) => {
+  return (
+    _.isEmpty(formValues.emailTo) ||
+    _.isEmpty(formValues.emailFrom) ||
+    _.isEmpty(formValues.emailSmarthost) ||
+    _.isEmpty(formValues.emailHello)
+  );
+};
+
+const updateConfig = (config, globals, formValues, saveAsGlobal = false) =>
+  // Only save SMTP config values to receiverConfig or global if formValue is defined and different from global property
+  // If they are the same, don't save in receiverConfig so the global property will be used
+  _.forEach(SMTP_CONFIG_FIELDS, (propName) => {
+    const formValue = getFormValue(formValues, FORM_PREFIX, propName);
+    const globalValue = getGlobalValue(globals, GLOBAL_PREFIX, propName);
+    if (notEqualAndNotEmpty(formValue, globalValue)) {
+      _.set(
+        config,
+        saveAsGlobal ? `smtp_${propName}` : propName,
+        formValue,
+        // TODO: does auth_secret need to be encoded?
```

Comment:
@kyoto Do you know if we need to Base64.encode auth_secret which is a a CRAM-MDS value type.

Reply:
@spadgett, any thoughts on ^^?  -thanks